src/docs/sdk/data-handling.mdx

```diff
@@ -51,6 +51,42 @@ Some examples of auto instrumentation that could attach sensitive data:
 - Desktop apps including window title.
 - A Web framework routing instrumentation attaching route `to` and `from`.
 
+## Structuring Data
+
+For better data scrubbing on the server side, SDKs should save data in a strucutured way, when possible. Starting point of the discussion was [RFC-0038](https://github.com/getsentry/rfcs/blob/main/text/0038-scrubbing-sensitive-data.md)
```

Comment:
Our of curiosity, how does this affect Relay? Does separating the URL in to the base URL, query string, and fragment make Relay's life easier somehow? How will it use this structured data?